client/client.go

```diff
@@ -54,6 +54,8 @@ type Client interface {
 	// Also it may return nil if PD finds no Region for the key temporarily,
 	// client should retry later.
 	GetRegion(ctx context.Context, key []byte) (*Region, error)
+	// GetRegionRandomly gets a region from the server which responds earliest
+	GetRegionRandomly(ctx context.Context, key []byte) (*Region, error)
```

Comment:
How about `GetRegionFromMember`,  we only need GetRegion from the specified member or specified members.

Reply:
> How about `GetRegionFromMember`, we only need GetRegion from the specified member or specified members.

But this method is to get region from the member which responds first, could we name it `GetRegionFromMember` ?

Reply:
Yes, I mean the function only get the region from the specified PD, rather than all PDs, and pick the first response.

Reply:
OK